tests/Microsoft.DotNet.Framework.Docker.Tests/WcfImageTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.Framework.Docker.Tests
+{
+    public class WcfImageTests
+    {
+        private static ImageDescriptor[] WcfTestData = new ImageDescriptor[]
+        {
+            new ImageDescriptor { Version = "4.6.2", OsVariant = OsVersion.WSC_LTSC2016 },
+            new ImageDescriptor { Version = "4.7", OsVariant = OsVersion.WSC_LTSC2016 },
+            new ImageDescriptor { Version = "4.7.1", OsVariant = OsVersion.WSC_LTSC2016 },
+            new ImageDescriptor { Version = "4.7.2", OsVariant = OsVersion.WSC_LTSC2016 },
+            new ImageDescriptor { Version = "4.7.2", OsVariant = OsVersion.WSC_1803 },
+            new ImageDescriptor { Version = "4.7.2", OsVariant = OsVersion.WSC_LTSC2019 },
+            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_1803 },
+            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_LTSC2016 },
+            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_LTSC2019 },
+            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_1903 },
+        };
+
+        private readonly ImageTestHelper imageTestHelper;
+
+        public WcfImageTests(ITestOutputHelper outputHelper)
+        {
+            imageTestHelper = new ImageTestHelper(outputHelper);
+        }
+
+        // Skip the test if it's for 3.5 to avoid empty MemberData (see https://github.com/xunit/xunit/issues/1113)
+        [SkippableTheory("3.5")]
+        [Trait("Category", "wcf")]
+        [MemberData(nameof(GetVerifyWcfImagesData))]
+        public void VerifyWcfImagesWithApps(ImageDescriptor imageDescriptor)
+        {
+            VerifyWcfImages(imageDescriptor);
```

Comment:
Given there is only one call to VerifyWcfImages, I think it would make the code more readable remove the method and inline the logic here.  The same comment applies to the ASP.NET tests